website/templates/include/profile/jobs.mako

```diff
@@ -96,7 +95,7 @@
             </div>
 
             <div>
-                <a class="btn btn-default" data-bind="click: addContent">
+                <a class="btn btn-default" data-bind="disable, click: addContent">
```

Comment:
disable is a knockout binding that disables a button if the validation requirements are not met.  In this case, if the job or school is empty.